SetCurrentVersion.cmd

```diff
@@ -1,4 +1,4 @@
 set MAJOR=1
-set MINOR=6
-set PATCH=0
+set MINOR=5
+set PATCH=3
```

Comment:
Took me four tries to get the patch correct. We need to preserve the latest version in our development branch until the last moment possible. Otherwise the 'delist' script won't function. We can fix this by also maintaining the last shipped version in this script, a change i will make soon.